components/generator/jvm/imagevector/src/main/kotlin/io/github/composegears/valkyrie/generator/jvm/imagevector/spec/BackingPropertySpec.kt

```diff
@@ -18,7 +18,7 @@ import io.github.composegears.valkyrie.ir.IrImageVector
 
 internal class BackingPropertySpec(private val config: ImageVectorSpecConfig) {
 
-    fun createAsBackingProperty(irVector: IrImageVector): ImageVectorSpecOutput {
+    fun createAsBackingProperty(irVector: IrImageVector): ImageVectorSpecOutput = with(config) {
         val backingProperty = backingPropertySpec(
             name = config.iconName.backingPropertyName(),
```

Comment:
You already in context of config
```suggestion
            name = iconName.backingPropertyName(),
```